Test1Rewrite/task2.c

```diff
@@ -0,0 +1,93 @@
+#include <stdio.h>
+#include <stdbool.h>
+#include <time.h>
+#include <stdlib.h>
+
+bool isSorted(int array[], int size);
+bool shuffle(int array[], int size);
+void swap(int *oneValue, int *anotherValue);
+void monkeySort(int array[], int size);
+bool tests();
+
+int main() {
+    if (!tests()) {
+        return 1;
+    }
+    
+    srand(time(0));
```

Comment:
```suggestion
    srand(time(NULL));
```
Это то же самое, но time указатель принимает